docs/extensions-plugins/cli/commands.mdx

```diff
@@ -72,13 +81,21 @@ Lists all accessible AI models.
 pieces list models
 ```
 
-### `open`
-Opens an asset from a list or search. If only "open" is used, it opens the most recent asset.
+### `open` (deprecated)
+Opens an asset from a list or search. If only "open" is used, it opens the most recent asset. This also creates a link to the asset's code.
 
 ```bash
 pieces open [ITEM_INDEX]
```

Comment:
Can we here give some examples on the editor_command?
I am not sure what is the editor_command means here in that context you can say eg: code to open vs code or subl to open sublime etc.....

Reply:
addressed in [71304ba](https://github.com/pieces-app/documentation/pull/538/commits/71304bafa867c867a2e262326dc18c29bbff1bf4) commit